Assets/HoloToolkit/Utilities/Scripts/Editor/CapabilitySettingsWindow.cs

```diff
@@ -30,6 +31,8 @@ protected override void ApplySettings()
             ApplySetting(PlayerSettings.WSACapability.SpatialPerception);
             ApplySetting(PlayerSettings.WSACapability.WebCam);
             ApplySetting(PlayerSettings.WSACapability.InternetClient);
+            ApplySetting(PlayerSettings.WSACapability.InternetClientServer);
+            ApplySetting(PlayerSettings.WSACapability.PrivateNetworkClientServer);
```

Comment:
What is this capability used for in the Toolkit that isn't covered by InternetClientServer?

Reply:
This goes back to our prev conversation in the update PR haha.

I'll apply those learnings.

Reply:
Sounds good!

Reply:
The [documentation for win 10 capabilities](https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-uap-capability) hasn't been updated recently. Keeps [sending me back to win 8](https://msdn.microsoft.com/library/windows/apps/hh464936).

Reply:
Weird, yeah, that's definitely a stale link. [Here's the Win10 documentation.](https://docs.microsoft.com/en-us/windows/uwp/devices-sensors/enable-device-capabilities)

Reply:
Weird, yeah, that's definitely a stale link. [Here's the Win10 documentation.](https://docs.microsoft.com/en-us/windows/uwp/packaging/app-capability-declarations)